style.puml

```diff
@@ -1,136 +1,131 @@
 @startuml
 
-!ifndef FONTNAME
-!define FONTNAME "Verdana"
-!endif
-
-!ifndef FONTSIZE
-!define FONTSIZE 11
-!endif
+!$FONTNAME = "Verdana"
```

Comment:
Please make sure you don't overwrite fonts predefined by users of the script (`ifndef` is deliberate), here is an example with the new syntax:
```plantuml
!if (%not(%variable_exists("$ab")))
!$ab = "foo1"
!endif```

Reply:
Updated